packages/migration-sdk-viem/src/config.ts

```diff
@@ -140,6 +140,13 @@ export const MIGRATION_ADDRESSES = {
     [MigratableProtocol.aaveV2]: null,
     [MigratableProtocol.aaveV3Optimizer]: null,
   },
+  [ChainId.PolygonMainnet]: {
```

Comment:
Why not making it optional by chain instead of specifying null addresses

Reply:
Because we declare it as `const` and use it as is ; we'd need a typed accessor